caldav/server.go

```diff
@@ -32,9 +30,9 @@ type PutCalendarObjectOptions struct {
 // Backend is a CalDAV server backend.
 type Backend interface {
 	CalendarHomeSetPath(ctx context.Context) (string, error)
-	Calendar(ctx context.Context) (*Calendar, error)
+	Calendars(ctx context.Context) ([]Calendar, error)
```

Comment:
For symmetry with calendar objects, maybe we could rename this to `ListCalendars`?